manual/src/html_processing/Makefile

```diff
@@ -8,8 +8,9 @@ else
     DBG=quiet
 endif
 
-WEBDIR = ../webman
-WEBDIRMAN = $(WEBDIR)/manual
+VERSION=$(shell grep 'OCAML_VERSION_SHORT=' ../../../configure | awk -F'=' '{print $$2}')
```

Comment:
The version number is available without shell invocation by including `Makefile.common` or `Makefile.config`.